spec.html

```diff
@@ -36466,22 +36456,12 @@ <h1>%AsyncFromSyncIteratorPrototype%.return ( _value_ )</h1>
               1. Let _iterResult_ be ! CreateIterResultObject(_value_, *true*).
               1. Perform ! Call(_promiseCapability_.[[Resolve]], *undefined*, &laquo; _iterResult_ &raquo;).
               1. Return _promiseCapability_.[[Promise]].
-            1. Let _returnResult_ be Call(_return_, _syncIterator_, &laquo; _value_ &raquo;).
-            1. IfAbruptRejectPromise(_returnResult_, _promiseCapability_).
-            1. If Type(_returnResult_) is not Object, then
+            1. Let _result_ be Call(_return_, _syncIterator_, &laquo; _value_ &raquo;).
+            1. IfAbruptRejectPromise(_result_, _promiseCapability_).
+            1. If Type(_result_) is not Object, then
               1. Perform ! Call(_promiseCapability_.[[Reject]], *undefined*, &laquo; a newly created *TypeError* object &raquo;).
               1. Return _promiseCapability_.[[Promise]].
-            1. Let _returnDone_ be IteratorComplete(_returnResult_).
-            1. IfAbruptRejectPromise(_returnDone_, _promiseCapability_).
-            1. Let _returnValue_ be IteratorValue(_returnResult_).
-            1. IfAbruptRejectPromise(_returnValue_, _promiseCapability_).
-            1. Let _valueWrapperCapability_ be ! NewPromiseCapability(%Promise%).
-            1. Perform ! Call(_valueWrapperCapability_.[[Resolve]], *undefined*, &laquo; _returnValue_ &raquo;).
-            1. Let _steps_ be the algorithm steps defined in <emu-xref href="#sec-async-from-sync-iterator-value-unwrap-functions" title></emu-xref>.
```

Comment:
@domenic can you confirm that this step is fine to move into an abstract operation?